src/sentry/static/sentry/app/views/organizationDashboard/index.jsx

```diff
@@ -47,6 +47,10 @@ class Dashboard extends React.Component {
 
     const hasTeamAdminAccess = access.has('team:admin');
 
+    if (projects.length === 1 && !projects[0].firstEvent) {
```

Comment:
Should we check if all projects don't have a first event and disregard the number of projects?